### PR TITLE
Raidboss: E9N timeline

### PR DESCRIPTION
**Pull request by JLGarber**
This wasn't too bad to put together, but it does look like there's more variance on this than I'm used to. Wide-Angle Phaser and Rejuvenating Balm just seem to drift slightly from pull to pull. None of it actually matters since the exact timing on those two mechanics isn't make-or-break for anyone, but it's annoying. My suspicion is that when she tethers the wall, there's a difference in animation time between "tethers front" and "tethers back". Everything else seems to be mostly fine.

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e9n.txt [hunk 1]
```diff
@@ -4,7 +4,64 @@
...
 
+# -ii 5626 5B18 55E1 55E3 5159 532F 55DB 55E5
+
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
 
-0 "Start"
-0.0 "--sync--" sync /:Engage!/ window 0,1
+0 "Start" sync /Engage!/ window 0,1
+# Unfortunately we can't sync to a first attack here,
+# as Cloud of Darkness uses an auto at random before starting her 3-second rotation.
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9n.txt:13]: Could sync to the starts casting line of Ground-Razing Particle Beam to start the timeline sooner, maybe?

**Review by quisquous — COMMENTED** [review]
That seems reasonable overall.  It looks like there's plenty of windows, so small drift seems fine.  How much drift are you talking about, mostly for my own curiosity?

**Comment by JLGarber** [comment]
Not truly awful, it's usually around 0.6-0.8 seconds, but I've seen it max at something like 1.2.

**Comment by quisquous** [comment]
Thanks!

**Pull request merged**